package.json

```diff
@@ -20,7 +20,7 @@
     "cover": "npm test -- --coverage",
     "start": "start-storybook -s ./static -p 8080",
     "build:docs": "build-storybook -s ./static -o ./build",
-    "build": "microbundle --generateTypes false --jsx React.createElement --compress false",
+    "build": "microbundle --generateTypes false --jsx React.createElement --compress false --globals react=React,react-dom=ReactDOM,prop-types=PropTypes,react-transition-group=ReactTransitionGroup,react-popper=ReactPopper,classnames=classNames",
```

Comment:
Would this affect importing the library without those globals being defined?

Reply:
I didn't understand the question, but the UMD build was not working without these globals. If you remove `react=React` from the above, then it'll throw `React__default.default is undefined`. In [this issue](https://github.com/reactstrap/reactstrap/issues/2552), it throws error for `propTypes`.